src/main/java/agh/ics/oop/gui/GuiElementBox.java

```diff
@@ -0,0 +1,79 @@
+package agh.ics.oop.gui;
+
+import agh.ics.oop.IMapElement;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+import javafx.scene.layout.VBox;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static agh.ics.oop.gui.App.CELL_SIZE;
+
+
+public class GuiElementBox {
+
+
+
+    private final boolean showLabel;
+    private final Map<String,ImageView> mapElementsView = new HashMap<>();
+
+
+    public GuiElementBox() {
+        this(false);
+    }
+
+    public GuiElementBox(boolean showLabel) {
+        this.showLabel = showLabel;
+    }
+
+    public VBox displayObject(IMapElement element){
+
+        String elementImageSrc = element.getImageSrc();
+
+        //nie wiem czemu to nie działa
```

Comment:
nie możemy użyć tej samej kontrolki jednocześnie w wielu miejscach - gdyby ta mapa była aktywna to moglibyśmy mieć tylko po jednym obrazku żółwia danego typu na planszy. Ogólnie cała klasa `GuiElementBox` w obecnym stanie bardziej przypomina fabrykę obiektów niż sam obiekt - chcieliśmy tu raczej mieć reprezentację pojedynczej  komórki siatki, która ma labelkę (jedną per komórka!) oraz obrazek (również jeden per komórka).